docs/tr2/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 - added a new easter egg command
 - added a `/lighting` console command to let the player turn lighting system on/off
 - added missing weapons to Lara's Home, Home Sweet Home and Nightmare in Vegas (for the weapons cheat) (#3360)
+- added an option to use TR1 snappy swim turn behaviour (#3387)
+- added an option to disable underwater twist, jump twist and swan-dive roll (#3388)
+- added an option to control responsive jumping lock behaviour (#3389)
```

Comment:
I'm thinking of introducing tighter control over changelog related to settings – can we outline how to reach these options in the UI? (Eg Gameplay settings → Fixes → [Name of the option])

Reply:
LMK if that looks OK, or if we'd prefer a different structure, maybe square brackets or something?